cl/tests/providers.py

```diff
@@ -80,3 +80,12 @@ def case_name(self, full: bool = False) -> str:
         plaintiff = self._make_random_party(full)
         defendant = self._make_random_party(full)
         return titlecase(f"{plaintiff} v. {defendant}")
+
+    def random_citation(self) -> str:
+        """Make a random citation e.g. 345 Mass. 76"""
+        volume = random.randint(1, 999)
+        page = random.randint(1, 999)
+        reporter = random.choice(
+            ["U.S.", "Mass.", "S.E. 2d", "MT", "Cal.", "Ohio"]
```

Comment:
For extra fun, can't you do:

```
random.choice(REPORTERS.keys())
```

(Or something similar that uses variations to make it even more realistic?)

Reply:
I do think that throws a little bit of chaos into the system.  

For example a neutral citation would fail if we landed on NMCA

Reply:
I do like the idea though so Im trying to figure out how to use it.